src/main/java/core/basesyntax/service/impl/ProcessServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProcessService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class ProcessServiceImpl implements ProcessService {
+    private final OperationStrategy operationStrategy;
+
+    public ProcessServiceImpl(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public Storage processObjects(List<FruitTransaction> data) {

```

Comment:
we can directly access Storage
```suggestion
    public void processObjects(List<FruitTransaction> data) {

```